presto-spi/src/main/java/com/facebook/presto/spi/block/FixedWidthBlockBuilder.java

```diff
@@ -92,7 +89,7 @@ public boolean isFull()
     @Override
     public int getSizeInBytes()
     {
-        long size = getRawSlice().length() + SizeOf.sizeOf(valueIsNull);
+        long size = getRawSlice().length() + valueIsNull.getUnderlyingSlice().length();
```

Comment:
this is different from what `getRawSlice` is doing, you are counting the entire underlying slice for the null slice